src/components/TransactionItem/TransactionItemCell/index.tsx

```diff
@@ -213,16 +215,21 @@ const TransactionCellUDT = ({ cell }: { cell: State.Cell }) => {
         <img src={UDTTokenIcon} className="transaction__cell__udt__icon" alt="udt token" />
       </Tooltip>
     </TransactionCellUDTPanel>
-  )
+  ) : null
 }
 
 const TransactionCellCapacity = ({ cell, cellType }: { cell: State.Cell; cellType: CellType }) => {
+  const { extraInfo } = cell
+  const udtExtraInfo = cell.cellType === 'udt' && extraInfo ? (extraInfo as State.UDTInfo) : null
```

Comment:
The type assertion can be avoided by a type guard

Ref: https://www.typescriptlang.org/docs/handbook/2/narrowing.html#using-type-predicates